scripts/Jenkinsfile

```diff
@@ -28,7 +28,12 @@ node {
                        '@LookupTableOptionSorting', '@ManualQuarantine', '@LogSubmission']
 
         if (custom_tag != '') {
-            tagArray =  [custom_tag]
+            if (custom_tag.indexOf(',') > 0) {
+                // Parse as a list if contains commas
+                tagArray = Eval.me(custom_tag)
```

Comment:
what does `Eval.me` do?

Reply:
Attempts to parse the String into different data type depending on the String format, so in this case if we pass in a String formatted like an Array (IE '@Tag', '@Tag2') it'll output an array with those values

Reply:
interesting, sounds magical